app/models/user.rb

```diff
@@ -1,2 +1,26 @@
+VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
 class User < ApplicationRecord
+  before_save :downcase_email
```

Comment:
```suggestion
class User < ApplicationRecord
  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i

  before_save :downcase_email
```
Đưa biến const vào trong class e nha